app/src/main/kotlin/com/wire/android/ui/calling/SharedCallingViewModel.kt

```diff
@@ -178,8 +188,24 @@ class SharedCallingViewModel @Inject constructor(
                 callState = callState.copy(
                     callStatus = call.status,
                     callerName = call.callerName,
-                    isMuted = call.isMuted,
                     isCameraOn = call.isCameraOn,
+                    isMuted = call.isMuted
```

Comment:
If the flow is filtered to only take `first` in this method, then we won't get any changes of these parameters during the call - is it intended?

Reply:
Good catch 👌